src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -31,6 +31,7 @@ public static class EnvironmentSettingNames
         public const string FunctionsExtensionVersion = "FUNCTIONS_EXTENSION_VERSION";
         public const string FunctionWorkerRuntime = "FUNCTIONS_WORKER_RUNTIME";
         public const string ContainerName = "CONTAINER_NAME";
+        public const string WebsitePodName = "WEBSITE_POD_NAME";
```

Comment:
For making the switch for CV2 from CONTAINER_NAME to WEBSITE_POD_NAME, are there other helpers we need to update other than just IsLinuxConsumptionOnLegion? Looking at all the usages of ContainerName currently, I see other places like GetInstanceId, etc. Will we be sending both, or will we only send one?

Reply:
We'll be sending both until we can completely decouple from `Container_Name`.